message tab_ShieldSkillTable 
{
	optional string tname = 1;
	repeated tab_ShieldSkill tlist = 2;
}

message tab_ShieldSkill
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 Level = 3;
	optional int32 InGameName = 4;
	optional int32 TrainCost = 5;
	optional int32 UpgradeCost = 6;
	optional int32 UpgradeTime = 7;
	optional int32 UpdateDiamond = 8;
	optional int32 DefenseAdd = 9;
}